### PR TITLE
Added Vertex Cover Problem

### PR DESCRIPTION
**Pull request by parasgoyal12**
Added the minimum vertex cover problem for binary trees. Please Review...

### CODE REVIEW

**Review by chandrikadeb7 — COMMENTED** [review]
Seems fine. Please mention the source in the comment after the code 

// Source: GeeksforGeeks (link)

**Comment by parasgoyal12** [comment]
Added the link. Please Review.

File: Task 1/Minimum Vertex Cover/Question.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Problem
+=====================
+Link to the problem : https://www.geeksforgeeks.org/vertex-cover-problem-set-2-dynamic-programming-solution-tree/
```
Comment (chandrikadeb7) [inline Task 1/Minimum Vertex Cover/Question.md:3]: Add Markdown syntax for program link. Rest is fine.

Comment (parasgoyal12) [reply Task 1/Minimum Vertex Cover/Question.md]: I have added the link in Markdown. Kindly review.

**Review by chandrikadeb7 — APPROVED** [review]

**Pull request merged**